lesson15/main.js

```diff
@@ -86,41 +84,52 @@ const displayModal= () => {
   overlay.style.display= "block";
 };
 
+const resetInput = () => {
+  const inputFields = document.getElementsByClassName("js-input");
+  const errorMessages = document.getElementsByClassName("js-error-message");
+
+  for (const errorMessage of errorMessages) {
+    errorMessage.textContent ="";
+  };
+  for(const inputField of inputFields) {
+    inputField.value = "";
+    inputField.classList.remove("error-border");
+  };
+}
+
 const closeModal = () => {
-  resetInput()
   modal.style.display="none";
   overlay.style.display="none";
+  resetInput();
 };
 
 const removeModal = () =>  document.getElementById("js-modal-wrapper").remove();
 
-const resetInput = () => {
-  numberInput.value = '';
-  validationMessage.style.display ="none";
-};
-
-const displayValidation = (message) => {
-  validationMessage.textContent = `${message}`;
-  validationMessage.style.display="block";
-}
-
-openListButton.addEventListener("click", () => {
-  const userInput = numberInput.value;
-  if(userInput === ''){
-    displayValidation("Input feild cannot be empty");
-    return;
-  }
-  if (!userInput.match(/^\d+$/)){
-    displayValidation("Input value must be a number");
-    return;
+const checkValidation = (inputArea, error, message) => {
+  if(inputArea.value === "") {
```

Comment:
nits
Since no error message appears even if you enter a space in the name input field,
I think it's better to use `trim()` here as well.
```suggestion
  if(inputArea.value.trim() === "") {
```

Reply:
I agree with Mai-san!